### PR TITLE
Feature/modal on hotspot tap

### PR DESCRIPTION
**Pull request by debreb**
Screenshot 2022-02-22 at 09 47 45

Screenshot 2022-02-22 at 09 45 26

### CODE REVIEW

File: screens/HotspotDetailScreen.tsx [hunk 1]
```diff
@@ -0,0 +1,281 @@
...
+      textAlign: 'center',
+    },
+  });
+
+export const HotspotDetailScreen = ({
+  route,
+}: NativeStackScreenProps<RootStackParamList, 'HotspotDetail'>) => {
+  const theme = useTheme();
+  const styles = getStyles(theme);
+  const hotspot = route.params.hotspot;
```
Comment (mveres) [inline screens/HotspotDetailScreen.tsx:147]: By looking at the server api, I think we can't use this.
From the server we can load either a list of hotspots with light data for each one (id, lat, long, status) or
each hotspot by `id` with all the details (address, volunteer, cats...)

So what we have to do here is pass the hotspot id via route params and use an make a request to load all the details for the hotspot.

You can use my PR as an example.

We can do this in another PR.

File: screens/HotspotDetailScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,281 @@
...
+
+  return (
+    <View style={styles.container}>
+      <ScrollView style={styles.scrollView}>
+        <>
```
Comment (mveres) [inline screens/HotspotDetailScreen.tsx:152]: Why is the fragment needed here?

File: screens/HotspotDetailScreen.tsx [hunk 3]
```diff
@@ -0,0 +1,281 @@
...
+          <View style={{ padding: 20 }}>
+            <View style={{ flexDirection: 'row', paddingTop: 18 }}>
+              <View style={{ width: '60%', justifyContent: 'flex-start' }}>
```
Comment (mveres) [inline screens/HotspotDetailScreen.tsx:155]: we should keep all the styles in the style object, not inline

**Review by mveres — CHANGES_REQUESTED** [review]

File: screens/HotspotDetailScreen.tsx (hunk 2 above)
Comment (debreb) [reply screens/HotspotDetailScreen.tsx:152]: it's deleted

File: screens/HotspotDetailScreen.tsx (hunk 1 above)
Comment (debreb) [reply screens/HotspotDetailScreen.tsx:147]: For now I leave this at it is, and do the changes in a separate branch.

Comment (debreb) [reply screens/HotspotDetailScreen.tsx]: I'll in a new branch

**Review by mveres — APPROVED** [review]

**Review by mveres — APPROVED** [review]
💯

**Pull request merged**